src/devices/bosch.ts

```diff
@@ -1505,8 +1531,17 @@ const definitions: Definition[] = [
         model: 'BMCT-SLZ',
         vendor: 'Bosch',
         description: 'Light/shutter control unit II',
-        fromZigbee: [fzLocal.bmct, fz.cover_position_tilt, fz.on_off, fz.power_on_behavior],
-        toZigbee: [tzLocal.bmct, tz.cover_position_tilt, tz.power_on_behavior],
+        fromZigbee: [
+            fz.on_off,
+            fz.power_on_behavior,
+            fz.cover_position_tilt,
+            fzLocal.bmct,
+        ],
+        toZigbee: [
+            tz.power_on_behavior,
+            tzLocal.bmct_cover_position_tilt,
+            tzLocal.bmct,
+        ],
         meta: {multiEndpoint: true},
```

Comment:
Instead of adding a new converter, I would use `meta: {multiEndpoint: true, coverPositionTiltDisableReport: true},` and use that in `tz.cover_position_tilt`